packages/batch-delegate/src/createBatchDelegateFn.ts

```diff
@@ -9,6 +9,7 @@ import { BatchDelegateOptionsFn, BatchDelegateFn, BatchDelegateOptions } from '.
 export function createBatchDelegateFn<K = any, V = any, C = K>(
   argFn: (args: ReadonlyArray<K>) => Record<string, any>,
   batchDelegateOptionsFn: BatchDelegateOptionsFn,
+  mappingFn?: (keys: ReadonlyArray<K>, results: any) => V[],
   dataLoaderOptions?: DataLoader.Options<K, V, C>
```

Comment:
In retrospect, this looks like a critical addition. However, to prevent a breaking change, this new option must be last, right?

Reply:
You're right. I figured the function intuitively made more sense in the third position, but this will break backwards compatibility. Maybe we can make the params a config object instead? Regardless, I updated the position to be last for now.